lntest/itest/lnd_test.go

```diff
@@ -11019,12 +11020,13 @@ func assertActiveHtlcs(nodes []*lntest.HarnessNode, payHashes ...[]byte) error {
 
 			// Make sure all the payHashes are active.
 			for _, payHash := range payHashes {
-				if _, ok := htlcHashes[string(payHash)]; ok {
+				h := hex.EncodeToString(payHash)
+				if _, ok := htlcHashes[h]; ok {
 					continue
 				}
 				return fmt.Errorf("node %x didn't have the "+
 					"payHash %v active", node.PubKey[:],
-					payHash)
+					h)
```

Comment:
can this commit be equally accomplished by replacing `%v` with `%x`? also doesn't incur the 2x memory overhead

Reply:
It would, but while debugging I found it useful to just spew the htlcHashed map, so nice to have them hex there IMO.

Reply:
seems like a simple change to make in the event one needs to debug, but it's not that important